pkgs/top-level/all-packages.nix

```diff
@@ -14197,6 +14197,8 @@ with pkgs;
     inherit (perlPackages.override { pkgs = pkgs // { imagemagick = imagemagickBig;}; }) PerlMagick;
   };
 
+  imagej = callPackage ../applications/graphics/imagej { };
```

Comment:
> What is the motivation for returning a set to begin with?

The reason I've used a set stems from my ignorance on a fundamental level; and on a superficial level, it is because jvm based nix packages I've consulted were popular giants like eclipse.